designs/administrator/4.6/tekton-triggers/index.md

```diff
@@ -0,0 +1,31 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Tekton Triggers
+
+## Adding Pipelines, Tasks, and Triggers pages under Pipelines
```

Comment:
Curious, what about Pipeline Resources? I don't see them being carried over to this design.

They are also in a different preview state than the other Pipeline Resources.

- Pipelines / Pipeline Runs / Tasks / Cluster Tasks / Task Runs are all *Tech Preview*
- Whereas Pipeline Resources are *Dev Preview*

Note: The above badge changes is of 0.11 (4.3 on `canary`) and 1.0 (4.4+) of the operator.